bird-slideshow.py

```diff
@@ -240,7 +240,7 @@ def download_img(cache_dir, img_link):
         return filepath
 
     except:
-        print("Error", f"Could not download {img_link}")
+        print("Error", "Could not download %s" % img_link)
```

Comment:
You could use a + here to avoid having to interpolate the string:

ex: "Could not download " + img_link

Reply:
%s interpolation is more consistent with the rest of the program.

Reply:
%s interpolation is more consistent with the rest of the program.